info.limpet.rcp/src/info/limpet/rcp/core_view/CoreAnalysisView.java

```diff
@@ -150,11 +150,10 @@ protected void newSelection(ISelection selection)
 					IStoreItem iC = iter.next();
 					iC.addChangeListener(changeListener);
 				}
-
-				// ok, display them
-				display(res);
 			}
 		}
+		// ok, display them
+		display(res);
 
```

Comment:
this display() call is in the wrong place.  It should only be called after a successful **appliesToMe()** call. Will move it.